media/public/include/IMediaKeys.h

```diff
@@ -35,6 +35,10 @@
 #include "IMediaKeysClient.h"
 #include "MediaCommon.h"
 
+namespace firebolt::rialto::client
```

Comment:
One of our project rules (which I don't like) tells to avoid forward declarations:
https://wiki.rdkcentral.com/display/ASP/Manual+Review